client/index.html

```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+﻿<!doctype html>
```

Comment:
it looks like I accidentally saved the file with a UTF-8 BOM identifier. I'll try to remove it in a mo